From 29ff3e85eac6f05865cbcb9ad13af05c0c0605ae Mon Sep 17 00:00:00 2001
From: Arun Ravindran <arun.ravindran@intel.com>
Date: Wed, 21 Nov 2012 16:01:22 +0200
Subject: [PATCH 05/38] bionic: Enabling telephony in JB 4.2

BZ: 47372

This patch fixes the compilation issues in stmd of missing
burst member variable.

Change-Id: I99eca417822ae49058898dafed233cc2ca1eec3e
Signed-off-by: Arun Ravindran <arun.ravindran@intel.com>
Reviewed-on: http://android.intel.com:8080/75248
Reviewed-by: Fremaux, Pascal <pascal.fremaux@intel.com>
Reviewed-by: Badrappan, Jeevaka <jeevaka.badrappan@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Lebsir, SamiX <samix.lebsir@intel.com>
Tested-by: Lebsir, SamiX <samix.lebsir@intel.com>
---
 libc/kernel/common/linux/gsmmux.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/libc/kernel/common/linux/gsmmux.h b/libc/kernel/common/linux/gsmmux.h
index 76185c5..e35a3ff 100644
--- a/libc/kernel/common/linux/gsmmux.h
+++ b/libc/kernel/common/linux/gsmmux.h
@@ -36,7 +36,8 @@ struct gsm_config {
  unsigned int i;
  unsigned int clocal;
 /* WARNING: DO NOT EDIT, AUTO-GENERATED CODE - SEE TOP FOR INSTRUCTIONS */
- unsigned int unused[7];
+ unsigned int burst;
+ unsigned int unused[6];
 };
 #define GSMIOC_GETCONF _IOR('G', 0, struct gsm_config)
 #define GSMIOC_SETCONF _IOW('G', 1, struct gsm_config)
-- 
1.7.9

